tests/integration_tests/io/io_json_detector_roundtrip.cpp

```diff
@@ -16,7 +16,7 @@
 #include "detray/io/frontend/detector_writer.hpp"
 #include "detray/io/json/json_reader.hpp"
 #include "detray/io/json/json_writer.hpp"
-#include "detray/test/toy_detector_test.hpp"
+#include "detray/test/cpu/toy_detector_test.hpp"
```

Comment:
I personally prefer `host` over `cpu`, but this I don't feel super strongly about...

Reply:
I don't feel strongly about it either, just kept it the way it is done in the rest of the code